Feature/UI optimize

seasonyuu opened the pull request:
主要修改内容：
1. 监听文件变化，对文件大小变化、修改时间变化、删除做出界面对应展示；
2. 表格 item 新增固定显示的修改日期项、定位文件按钮。
3. Windows 版本编译时，内嵌 WebView2

Commit:
Reveal file in partition table

Commit:
Show file modified time at partition table

Commit:
Fix reveal file on Windows

Commit:
Watch for file changes

Signed-off-by: xychen <xychen@listenai.com>

seasonyuu commented:
附图
![image](https://github.com/user-attachments/assets/e8fcf94d-8d57-4fde-8d1d-479e4ce9abe6)
![image](https://github.com/user-attachments/assets/2d205b5a-4ba1-490a-a52f-311a8a0b4a98)

xingrz commented on src/components/common/FormatTime.vue:
```diff
@@ -0,0 +1,31 @@
+}>();
+
+const date = new Date(props.datetime);
+const formatter = Intl.DateTimeFormat('zh-CN', {
```
不要自己造轮子，看看naive-ui有没有自带了dayjs或者什么东西。没有就弄个luxon吧

seasonyuu replied:
你说得没毛病，去掉了。  35c26639a9f5d4ba34fb29f51e933f902d3216be

xingrz commented on src-tauri/Cargo.toml:
```diff
@@ -15,7 +15,7 @@ tauri-build = { version = "2.0.0-beta", features = [] }
 [dependencies]
 tauri = { version = "2.0.0-beta", features = [] }
```
不如顺便帮我升个2.0正式版？

seasonyuu replied:
先到 2.1.1 。这里是因为 Tauri 的主仓到 2.1.1，但是 plugin 相关的版本到了 2.2.0 ，为了避免不必要的意外，先到 2.1.1 。
42cecbfa75cdd9e5293d115c57f68a603f05f3de

xingrz commented on src/components/common/FormatTime.vue:
```diff
@@ -0,0 +1,31 @@
+});
+</script>
+
+<style lang="scss" module>
```
多余的

Commit:
Embed the WebView2 Bootstrapper for Windows

Commit:
Upgrade to Tauri v2 release

Signed-off-by: xychen <xychen@listenai.com>

Commit:
Fix reveal file on Linux

All we need actually is `$DBUS_SESSION_BUS_ADDRESS`, which we can easily
achieved by running `dbus-send` via shell.

Commit:
Bump up vue-tsc to fix vulnerability

Commit:
Decode outputs of cskburn in queue

Also fix a case where multiple lines may arrived in one data event, which
leads `onPartition` not getting called.

Signed-off-by: xychen <xychen@listenai.com>

Commit:
Update deps to fix npm audit

Signed-off-by: xychen <xychen@listenai.com>

Commit:
Instantiate File return from Rust with `class-transformer`

Signed-off-by: xychen <xychen@listenai.com>

Commit:
Update serialport and deps

* needed by the following fs watch feature

Signed-off-by: xychen <xychen@listenai.com>
